src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -697,6 +697,137 @@ By placing a few extra blocks and changing their layout, you can create interest
 
 image::compose-configuration/page-builder-example.png[role="data-zoomable"]
 
+== Page block Duplication
+
+Page blocks can be duplicated, which makes it a breeze to reuse a page block.
+This saves you the hassle of having to recreate a page block every time you need to reuse one.
+
+==== Copying a page block
+
+A great use case for copying a page block is when you want to use it on a different page in the same namespace or even a different namespace!
+
+For copying, all you have to do is navigate to the page that holds the page block you are interested in, click the button highlighted below and the page block will be copied to your clipboard.
+In the image below, the title of the pages have been highlighted to indicate that they are different pages.
+
+.Copying a page block
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-block-copy.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "annotations": [
+    {
+      "x": 275,
+      "y": 16,
+      "w": 362,
+      "h": 36,
+      "kind": "box"
+    },
+    {
+      "x": 115,
+      "y": 373,
+      "w": 15,
+      "h": 16,
+      "kind": "box"
+    }
+  ]
+}
+----
+
+You can then navigate to any other page to reuse this page block as many times as you want by pasting it with kbd:[Ctrl+V] just as you would any other content on your clipboard.
+
+.Pasting a page block
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-block-paste.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "annotations": [
+    {
+      "x": 275,
+      "y": 10,
+      "w": 378,
+      "h": 50,
+      "kind": "box"
+    }
```

Comment:
The annotation is a bit off

![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/07ace707-88f1-485c-a1b7-4fd4aa37364c)